src/interventions/chrome-ios-62-63.js

```diff
@@ -0,0 +1,147 @@
+/**
+Copyright 2017 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/**
+ * README
+ *
+ * This intervention handles the scenario where Chrome cannot handle a number
+ * passed as the PaymentCurrencyAmount.value for iOS.
+ *
+ * https://bugs.chromium.org/p/chromium/issues/detail?id=785429
+ *
+ * FIX
+ *
+ * This code checks the UserAgent to see if the current environment is Chrome
+ * for iOS version 62 or 63 and if it is converts the numbers in all
+ * PaymentCurrencyAmount.value instances to strings before passing the objects
+ * to the browser.
+ */
+
+/**
+ * Converts a PaymentCurrencyAmount.value from number to string, if any.
+ * @param {(window.PaymentCurrencyAmount|undefined)} amount
+ */
+function convertPaymentCurrencyAmount(amount) {
+  if (!amount) {
+    return;
+  }
+
+  // Convert the value to String if it isn't one already.
+  amount.value = String(amount.value);
+}
+
+/**
+ * Converts PaymentCurrencyAmount.value instances from number to string inside
+ * instances of PaymentItem, if any.
+ * @param {(Array<!window.PaymentItem>|undefined)} displayItems
+ */
+function convertDisplayItems(displayItems) {
+  if (!displayItems || !(displayItems instanceof Array)) {
+    return;
+  }
+
+  for (let i = 0; i < displayItems.length; i++) {
+    convertPaymentCurrencyAmount(displayItems[i].amount);
+  }
+}
+
+/**
+ * Converts PaymentCurrencyAmount.value instances from number to string inside
+ * instances of PaymentShippingOption, if any.
+ * @param {(Array<!window.PaymentShippingOption>|undefined)} shippingOptions
+ */
+function convertShippingOptions(shippingOptions) {
```

Comment:
This function is identical to the one above, but has different variable names. Let's use the same function, e.g., `convertDisplayItemsOrShippingOptions()`.

Reply:
done.